### ISSUE TITLE
Use Symfony Flex

### ISSUE THREAD

**Issue by JavierCane** [CodelyTV/cqrs-ddd-php-example#73]
https://symfony.com/doc/current/setup/flex.html#using-symfony-flex-in-new-applications

**Comment by JavierCane** [comment]
Discarded. Despite of being something doable, we would have to customize the `config-dir` parameter for Symfony Flex (https://github.com/symfony/flex/pull/242).

However, this means we could only have 1 single `config-dir` (the one set on the `composer.json->extra->config-dir`). So we would have to go for having multiple `composer.json` files (one for each application).

Since this is something we want to avoid because of not having to deal with the dependencies of each applications (they're the same) while updating them.

So in conclusion, no Symfony Flex until they support multi-application repositories 🤷‍♂️
